doc/source/methods/utility_functions.rst

```diff
@@ -0,0 +1,15 @@
+.. _ref_utility_functions:
+
+Utility functions
+==================
+
+.. currentmodule:: ansys.motorcad.core.rpc_client_core
+
+
+.. autosummary::
+   :toctree: _autosummary_utility_functions
+
+   MotorCADError
```

Comment:
This isn't a function - it's an class/exception type. Not sure it should be here. 

It does need to be documented somewhere. It would be good to add some more info to its docstring and maybe an example? (https://numpydoc.readthedocs.io/en/latest/format.html#examples)